bigtable/client/mutations.h

```diff
@@ -39,6 +39,12 @@ struct Mutation {
 Mutation SetCell(std::string family, std::string column, std::int64_t timestamp,
                  std::string value);
 
+/// Create a mutation to set a cell value where the server sets the time.
+Mutation SetCell(std::string family, std::string column, std::string value);
+
+/// A magic value where the server sets the timestamp.
```

Comment:
Can you note here that, if used, the mutation will not be idempotent and therefore retried?

Reply:
Done.

Reply:
I meant this for the `ServerSetTimestamp` function:

/// A magic value where the server sets the timestamp (which makes it non-idempotent blah blah blah).
 +constexpr std::int64_t ServerSetTimestamp() { return -1; }